packages/sdk/src/lib/dataProtectorCore/processProtectedData.ts

```diff
@@ -154,10 +159,13 @@ export const processProtectedData = async ({
       },
     });
     const requestorder = await iexec.order.signRequestorder(requestorderToSign);
-    const { dealid, txHash } = await iexec.order.matchOrders({
-      requestorder,
-      ...underMaxPriceOrders,
-    });
+    const { dealid, txHash } = await iexec.order.matchOrders(
+      {
+        requestorder,
+        ...underMaxPriceOrders,
+      },
+      { preflightCheck: false, useVoucher: vUseVoucher }
```

Comment:
wy adding `preflightCheck: false`?
disabling preflightCheck can create bad deals that produce always falling tasks.

```suggestion
      { useVoucher: vUseVoucher }
```